src/native/libs/System.Globalization.Native/CMakeLists.txt

```diff
@@ -150,9 +151,6 @@ if(CLR_CMAKE_TARGET_WIN32)
         )
         target_compile_options(System.Globalization.Native.Aot.GuardCF PRIVATE /GL-)
 
-        target_link_libraries(System.Globalization.Native.Aot ${__LinkLibraries})
```

Comment:
Should we remove the definition if it will no longer be used anywhere in the repo? https://github.com/dotnet/runtime/blob/1041cd93001e37ed7efbcea5e09474d574629427/src/native/libs/build-native.cmd#L20